lib/ofx/parser/signon.ex

```diff
@@ -34,4 +37,12 @@ defmodule Ofx.Parser.Signon do
     do: Map.put(messages, :signon, signon)
 
   defp get(xml, expression), do: SweetXml.xpath(xml, expression)
+
+  defp get_and_format_date(xml, expression) do
+    xml
+    |> get(expression)
+    |> Datetime.format()
+  rescue
+    _error -> nil
+  end
```

Comment:
O rescue vai esconder qualquer erro de formatação que possa ocorrer.

```suggestion
  defp get_and_format_date(xml, expression) do
   
   case get(xml, expression) do
       "" -> nil
       date -> Datetime.format(date)
    end
  end
```

Reply:
sim, infelizmente estamos sendo obrigados a isso, o Bradesco manda uma data inválida ("000000000000") e não temos garantia de que outros que fazem o mesmo vão mandar o mesmo valor, deixei intencionalmente engolir o erro assumindo que essa é uma informação "opcional", o que acha?

Reply:
😭 

Bom, talvez então só mudar o nome da fulção `get_and_format_date` para `get_and_format_export_date` pra ficar claro que esse não é um comportamento default para qualquer data, só para a data do export.